File: abc/abc331/a.java
// https://atcoder.jp/contests/abc331/submissions/48169181

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int M = stdIn.nextInt();
    int D = stdIn.nextInt();

    int y = stdIn.nextInt();
    int m = stdIn.nextInt();
    int d = stdIn.nextInt();

    if(m == M && d == D) {
      System.out.printf("%d %d %d", y + 1, 1, 1);
    } else if(d == D) {
      System.out.printf("%d %d %d", y, m + 1, 1);
    } else {
      System.out.printf("%d %d %d", y, m, d + 1);
    }
  }
}

File: abc/abc331/a.cs
// https://atcoder.jp/contests/abc331/submissions/48169403

using static System.Console;

public class A
{
  static void Main()
  {
    int[] t = ReadLine().Split().Select(int.Parse).ToArray();
    int M = t[0];
    int D = t[1];

    t = ReadLine().Split().Select(int.Parse).ToArray();
    int y = t[0];
    int m = t[1];
    int d = t[2];

    if(m == M && d == D) {
      Write($"{y + 1} {1} {1}");
    } else if(d == D) {
      Write($"{y} {m + 1} {1}");
    } else {
      Write($"{y} {m} {d + 1}");
    }
  }
}

